Minio.nuspec

```diff
@@ -1,35 +0,0 @@
-<?xml version="1.0"?>
-<package >
-  <metadata>
-    <id>Minio</id>
-    <version>1.1.2</version>
-    <authors>Minio, Inc.</authors>
-    <owners>Minio, Inc</owners>
-    <licenseUrl>http://www.apache.org/licenses/LICENSE-2.0.html</licenseUrl>
-    <projectUrl>https://github.com/minio/minio-dotnet</projectUrl>
-    <iconUrl>https://www.minio.io/logo/</iconUrl>
-    <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <description>Minio .NET SDK for Amazon S3 Compatible Cloud Storage</description>
-    <releaseNotes>Rewrite of .NET sdk for support of .NetCore and .NET4.5.2 framework</releaseNotes>
-    <copyright>Copyright 2017</copyright>
-    <tags>minio cloud storage</tags>
-    <dependencies>
-     <group targetFramework="net452">
-        <dependency id="RestSharp" version="105.2.3" />
-        <dependency id="Newtonsoft.Json" version="10.0.2" />
-        <dependency id="System.Reactive" version="3.1.1" />
-        <dependency id="System.Reactive.Core" version="3.1.1" />
-        <dependency id="System.Reactive.Interfaces" version="3.1.1" />
-        <dependency id="System.Reactive.Linq" version="3.1.1" />
-        <dependency id="System.Reactive.PlatformServices" version="3.1.1" />
-        <dependency id="System.Reactive.Windows.Threading" version="3.1.1" />
-        <dependency id="System.Xml.Linq" version="3.5.21022.801" />
-      </group>
-   
-    </dependencies>
-    
-  </metadata>
-  <files>
-      <file src="Minio.Net452\bin\Release.Net452\**" target="lib" />
-  </files>
-</package>
```

Comment:
@niemyjski, MInio.nuspec is needed for creating nuget package by our automated release system. Can you add this file back with updated version, dependencies and target framework

Reply:
No, these are not needed, the new build system will automatically create packages for you. We shouldn't reinvent the wheel here. The build is creating nuget packages that match the previous system with tons of deleted code == easier to maintain.